Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:lts-alpine
+
+COPY . /app
+
+WORKDIR /app
+
+ENV PATH "$PATH:/app/node_modules/.bin"
+
+RUN npm install
```

Comment:
What do you think about using https://docs.npmjs.com/cli/ci.html here?

Reply:
Cool! I wasn't aware of that feature :+1: